apps/jsonrpc2/lib/jsonrpc2/spec_handler.ex

```diff
@@ -1,5 +1,127 @@
 defmodule JSONRPC2.SpecHandler do
   use JSONRPC2.Server.Handler
 
-  def handle_request("web3_clientVersion", _), do: Application.get_env(:jsonrpc2, :mana_version)
+  alias Blockchain.Block
+  alias Blockchain.Blocktree
+  alias ExthCrypto.Hash.Keccak
+  alias ExthCrypto.Math
+  alias ExWire.PeerSupervisor
+  alias ExWire.Sync
+  # TODO split methods into separate modules per topic
+  # web3 Methods
+
+  def handle_request("web3_clientVersion", _),
+    do: "#{Application.get_env(:jsonrpc2, :mana_version)}"
```

Comment:
I think string interpolation is not needed here

Reply:
I think you’re right!